tests/test_cmfd_feed/results_true.dat

```diff
@@ -124,6 +124,10 @@ tally 3:
 1.020705E+00
 5.413570E-02
 tally 4:
+3.049469E+00
```

Comment:
Why is there additional info in the CMFD test results given your changes to the surface tallies? Sorry for my ignorance, I didn't review any of the Fortran diff yet since I'm not familiar with that part of the codebase.